### PR TITLE
Batch delete

### LINKED ISSUE #2703: Desktop: Deleting Notebook Slower Than Selecting All Notes and Hitting Delete
## Environment

Joplin version: all versions past v1.0.179 (don't know about previous versions) 
Platform: Linux
OS specifics: Artix Linux 64 bit

## Steps to reproduce

1. Create a new notebook (any name will do)
2. Create 4 different notes
3. Duplicate these notes as many times as you can (i did over 1000 in my test)
4. Select 1st note 
5. Hold Shift and select last note
6. Hit Delete key or Click Delete on Right 
7. Click Yes 
8. Repeat Steps 1-3 
9. Select Notebook 
10. Repeat Step 6

## Describe what you expected to happen
Both should delete notes at the same speed but Notebook delete is drastically slower. 

## Logfile

Not home but can produce a log when i arrive.

### PR DESCRIPTION
**Pull request by RedDocMD**
Fixes #2703

### CODE REVIEW

**Comment by tessus** [comment]
This means your PR must only include one file: `ReactNativeClient/lib/models/Folder.js`

File: ReactNativeClient/lib/models/Folder.js [hunk 1]
```diff
@@ -68,9 +68,10 @@ class Folder extends BaseItem {
 
 		if (options.deleteChildren) {
 			let noteIds = await Folder.noteIds(folderId);
-			for (let i = 0; i < noteIds.length; i++) {
-				await Note.delete(noteIds[i]);
-			}
+			// for (let i = 0; i < noteIds.length; i++) {
+			// 	await Note.delete(noteIds[i]);
+			// }
```
Comment (tessus) [inline ReactNativeClient/lib/models/Folder.js]: remove

**Comment by tessus** [comment]
Now please remove the first 2 commits.

keyword `interactive rebase`

File: ReactNativeClient/lib/models/Folder.js (hunk 1 above)
Comment (RedDocMD) [reply ReactNativeClient/lib/models/Folder.js]: I am closing this PR and making a fresh one

Comment (tessus) [reply ReactNativeClient/lib/models/Folder.js]: For future development I'd suggest to learn git. git is more than `git pull` and `git push`.

**Pull request closed without merging**